README.md

```diff
@@ -2,6 +2,8 @@
 
```

Comment:
🖼️ Great addition of the OtterAI image! This enhances the visual appeal of the README. 🎨